index.css

```diff
@@ -0,0 +1,352 @@
+:root {
+  font-size: 62.5%;
+  box-sizing: border-box;
+}
+.wrapper {
+  position: relative;
+  width: 100%;
+  min-height: 38rem;
+
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  flex-grow: 1;
+}
+
+.round-image {
+  position: absolute;
+  top: 2rem;
+  right: -3.4rem;
+  width: 100%;
+}
+.round {
+  position: absolute;
+  top: 0rem;
+  right: 0rem;
+  width: 8rem;
+  height: 8rem;
+  border-radius: 50%;
+  border: none;
+  outline: none;
+  object-fit: cover;
+}
+.card-img img {
+  width: 100%;
+  height: 40rem;
+}
+.cards-example {
+  position: relative;
+
+  width: 35rem;
+}
+
+/* card with exit button */
+
+.card-exit {
+  display: flex;
+  flex-direction: column;
+
+  position: relative;
+  width: 30rem;
+
+  box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
+}
+.exit-button {
+  position: absolute;
+  top: 0;
+  right: 0;
+  padding: 1rem;
+  margin-right: 0.5rem;
+  border: none;
+  margin-top: 0.5rem;
+  background-color: white;
+  border-radius: 50%;
+  z-index: 1;
+}
+.cross-image {
+  max-height: 30rem;
+  object-fit: fill;
+}
+.exit-button:hover {
+  background: cadetblue;
+}
+.card-img img {
+  width: 100%;
+  height: 30rem;
+}
+.cart-button {
+  padding: 0.8rem;
+  /* width:10rem;
+          height:3rem; */
+
+  border: none;
+  outline: none;
+  background: black;
+  color: cornsilk;
+}
+.cart {
+  margin: 1rem;
+  padding: 1rem;
+}
+
+.card-body,
+.original-price,
+.discount-price,
+.price {
+  font-size: 2rem;
+}
+
+/* button component */
+
+.button-container-button {
+  padding: 1rem 2rem;
+  margin: 1rem;
+  /* margin-left:-0.3rem; */
+  display: inline-block;
+  width: 15rem;
+  font-weight: 800;
+  font-size: 1.8rem;
+  border: none;
+  border-radius: 1rem 1rem 1rem 1rem;
```

Comment:
Suggestion: Here too, try using the shorthand property if you want to have same border-radius for all corners